CheckBox: update demo description (#2019)
DevExpress/devextreme-demos: JSDemos/Demos/CheckBox/Overview/Angular/description.md

> When you add the CheckBox component to an application, specify its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can have the following states: 

Reviewer: ```suggestion
When you add the CheckBox component to an application, specify its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to set its state. The CheckBox can have one of the following states: 
```

- Checked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `true`).
- Unchecked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `false`).
- Indeterminate ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `null` or `undefined`).

> The component supports three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through this CheckBox states in the following order:

Reviewer 2: ```suggestion
The component supports the three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through the states of this CheckBox in the following order:
```

> The component supports the three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through the states of this CheckBox in the following order:

Reviewer: ```suggestion
The component supports the three states mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three states mode" CheckBox is enabled. You can cycle through the states of this CheckBox in the following order:
```

*Indeterminate → Checked → Unchecked → Indeterminate →  ...*

> To handle value change, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value change, for example, if you need to use the previous value.

Reviewer 2: > use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property.

To which component property should I bind the Value property?

```suggestion
To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value changes. For example, this scenario can be useful if you need to use the previous value.
```
Please do not overcomplicate sentences.

Author: > To which component property should I bind the Value property?

It depends on a property you want to change while checking the checkbox. In the demo, the checkbox **value** is bound to another checkbox **value**, so when you check the first, the second is checked too simultaneously. But the Checkbox **value** may be bound to any other component property, it can enable/disable Chart export, show/hide Popup, etc. That's why I write "**a** component property".

Maybe this version is better:

> To handle value changes, use two-way binding of the **value** property to a component property. The value of the component property depends on the **value** of the CheckBox.

Or even this:

> To handle value changes, use two-way binding. Bind the **value** property to a component property that you want to change when users click the CheckBox.

Reviewer 2: Yeah, now it has more sense. Thank you.

> To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property. The value of the component property depends on the value of the CheckBox. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value changes. For example, this technique can be useful if you need to use the previous value.

> You can use the [iconSize](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#iconSize) property to specify a custom width and height for the CheckBox. To add a label to the CheckBox, set the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) property.

Reviewer 2: ```suggestion
You can use the [iconSize](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#iconSize) property to specify custom dimensions for the CheckBox. To add a label to the CheckBox, set the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) property.
```
